src/apps/companies/apps/exports/client/ExportIndex.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Details from '@govuk-react/details'
+import Link from '@govuk-react/link'
+import { H3 } from '@govuk-react/heading'
+import Button from '@govuk-react/button'
+import { GREY_3, BLACK } from 'govuk-colours'
+import { SummaryTable } from 'data-hub-components'
+
+const NoTopMarginSummaryTable = styled(SummaryTable)`
+  margin-top: 0;
+`
+
+const ExportIndex = ({
+  isArchived,
+  exportDetails,
+  exportCountriesInformation,
+  exportPotentials,
+  findExportersUrl,
+  supportUrl,
+  exportWinsUrl,
+  editCountriesUrl,
+}) => {
+  return (
+    <>
+      <SummaryTable caption="Exports">
+        {exportDetails.map((row) =>
+          row.value.url ? (
```

Comment:
This condition could be within `SummaryTable.Row`, e.g.

```suggestion
          <SummaryTable.Row heading={row.heading} key={row.heading}>
            {row.value.url ? <span><Link>...</Link></span> : row.value}
          </SummaryTable.Row>
```